examples/example_exports/export_rms_data/export_preprocessed_surface.py

```diff
@@ -0,0 +1,46 @@
+"""Export metadata for a preprocessed surfaces via dataio."""
+
+import logging
+from pathlib import Path
+
+import xtgeo
+
+from fmu.config import utilities as utils
+from fmu.dataio.dataio import ExportData
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.WARNING)
+
+CFG = utils.yaml_load("../../fmuconfig/output/global_variables.yml")
+
+FILES = {
+    "depth": Path("../output/maps/structure/") / "topvolantis--ds_extract_geogrid.gri",
+}
```

Comment:
```suggestion
FILE = Path("../output/maps/structure/") / "topvolantis--ds_extract_geogrid.gri"
```

This dictionary form is a bit confusing, probably it was intended to use the key as the `content` and extend this to export multiple files. But should we rather change it for simplicity? and do:  `surf_depth = xtgeo.surface_from_file(FILE)`

Reply:
Totally agree. Thanks